K6/script.js

```diff
@@ -1,6 +1,26 @@
 import http from 'k6/http';
-import { sleep } from 'k6';
+import { check } from 'k6';
+
+export const options = {
+  scenarios: {
+    reviews_request: {
+      executor: 'constant-arrival-rate',
+      rate: 1000,
+      duration: '1m',
+      preAllocatedVUs: 80,
+      maxVUs: 100,
+    },
+  },
+};
+
+
 export default function () {
-  http.get('http://test.k6.io');
-  sleep(1);
```

Comment:
removing this command defaults you to sleep(0) which means there are no delays between requests